packages/react-storage/src/styles/storage-browser.css

```diff
@@ -39,6 +39,7 @@
 }
 .storage-browser__navigate__item {
   display: flex;
+  align-items: center;
   gap: var(--storage-browser-gap);
```

Comment:
Do you think the gap might be a little wide?

Reply:
Yeah, it looks a little weird. I need to do another pass at our grid styling so I can clean it up then.